lib/workos/directory_group.rb

```diff
@@ -5,17 +5,19 @@ module WorkOS
   # The DirectoryGroup class provides a lightweight wrapper around
   # a WorkOS DirectoryGroup resource. This class is not meant to be instantiated
   # in user space, and is instantiated internally but exposed.
-  class DirectoryGroup
+  class DirectoryGroup < Hash
     extend T::Sig
 
-    attr_accessor :id, :name
+    attr_accessor :id, :name, :custom_attributes, :raw_attributes
```

Comment:
I noticed that `DirectoryUser` exposes `:custom_attributes` and `:raw_attributes` through `attr_accessor`, so adding them here as well for consistency. I was originally planning to exclude these two from the hash, but since they're available for access on `DirectoryUser`, I figured it would be safer to keep them (and add them to `DirectoryGroup` for consistency).